### DISCUSSION TITLE
中文参考文献转化为相同标准的英文参考文献

### DISCUSSION THREAD

**Discussion by MrGongqi** [zotero-chinese/wiki#299] [求助 / Q&A]
你好，我在撰写一篇英文期刊的时候引用了十几个中文文献，请问有没有简洁的方法将中文参考文献转化成相同格式的英文参考文献呢？

**Comment by northword** [comment]
没有，另外，不建议在英文期刊论文中引用中文文献，绝大部分期刊出版社的系统都会验证参考文献的有效性，由于中文文献没有集中且有效的 DOI 管理，所以几乎所有的中文文献都无法通过验证。

**Comment by MrGongqi** [comment]
好的，我也是想到可能是这样的。6那我直接将中文删掉换成差不多的英文文献吧。谢谢啦
